### ISSUE TITLE
Handle new action "OPEN_TUNNEL"

### ISSUE THREAD

**Issue by tructhienbui** [MsLolita/grass#108]
They just present new action to the websocket call OPEN_TUNNEL.

This also make the script crashing with exception include "method" this is because the request_data does not contain the method any more.

I think that they change the regular HTTP_REQUEST action into another tunnel connection.

`09:47:24.46 INFO 1 | Connected
{'id': 'KJI2JJLFCroN6NpFXfhDr', 'action': 'OPEN_TUNNEL', 'data': {'url': 'https://api.getgrass.io/somethign/something'}}
{'url': 'https://api.getgrass.io/something/something'}

**Comment by milkthecow-moo** [comment]
The initial fix doesn't seem to hard. It appears to return the same data as the url in "HTTP" request which looks like this 

`{"code":"'abcdefghijk'"}`

The big question is what does the server expect to see?
with "HTTP_REQUEST" the server was expecting somthing like this from the client

`{
    "id": "12345",
    "origin_action": "HTTP_REQUEST",
    "result": {
        "body": "example_encoded_body_as_str",
        "headers": {
            "Content-Type": "application/json",
            "Cache-Control": "no-cache"
        },
        "status": 200,
        "status_text": "OK",
        "url": "https://example.com/api"
    }
}`

With OPEN_TUNNEL the server could be expecting something different

**Comment by tructhienbui** [comment]
Yeah i tried to debug on extensions but seem they dont return OPEN TUNNEL for extensions, so basically be careful. Only way to know is debug the desktop app it-self

**Comment by milkthecow-moo** [comment]
Yeah and that is hard. The app has SSL pining and I'm not knowledgeable enough to know how to remove it

**Comment by FiNT222** [comment]
Haven't found a solution to the problem yet?

**Comment by xingzai163** [comment]
I tried to capture the package, but the desktop could not find the websocket package

**Comment by Natmat626** [comment]
Haven't found a solution to the problem yet? Still waiting for it.
